searx/settings.yml

```diff
@@ -2,6 +2,7 @@ general:
   debug: false              # Debug mode, only for development
   instance_name: "SearXNG"  # displayed name
   contact_url: false        # mailto:contact@example.com
+  enable_metrics: false        # record stats
```

Comment:
The old behaviour was _true_ / "True" is also set as default in the settings_defaults.py

```diff
@@ -2,7 +2,7 @@ general:
   debug: false              # Debug mode, only for development
   instance_name: "SearXNG"  # displayed name
   contact_url: false        # mailto:contact@example.com
-  enable_metrics: false        # record stats
+  enable_metrics: true      # record stats
```

Reply:
Fixed